src/botPage/view/View.js

```diff
@@ -153,38 +154,45 @@ export default class View {
     constructor() {
         logHandler();
         this.initPromise = new Promise(resolve => {
-            symbolPromise.then(() => {
-                updateTokenList();
-                this.blockly = new _Blockly();
-                this.blockly.initPromise.then(() => {
-                    this.setElementActions();
-                    $('#accountLis');
-                    startRealityCheck(
-                        null,
-                        $('.account-id')
-                            .first()
-                            .attr('value')
-                    );
-                    ReactDOM.render(<Tour />, $('#tour')[0]);
-                    ReactDOM.render(
-                        <OfficialVersionWarning
-                            show={
-                                !(
-                                    typeof window.location !== 'undefined' &&
-                                    window.location.host === 'bot.binary.com' &&
-                                    window.location.pathname === '/bot.html'
-                                )
-                            }
-                        />,
-                        $('#footer')[0]
-                    );
-                    ReactDOM.render(<TradeInfoPanel />, $('#summaryPanel')[0]);
-                    ReactDOM.render(<LogTable />, $('#logTable')[0]);
-                    resolve();
+            updateConfigCurrencies().then(() => {
+                symbolPromise.then(() => {
+                    updateTokenList();
+                    this.blockly = new _Blockly();
+                    this.blockly.initPromise.then(() => {
+                        this.setElementActions();
+                        this.constructor.initRealityCheck();
+                        this.constructor.renderReactComponents();
+                        resolve();
+                    });
                 });
             });
         });
     }
+    static initRealityCheck() {
```

Comment:
Why static? Constructor is able to run these even when they're not static.

Reply:
Constructors are exempt from the rule.